### PR TITLE
Added updates on Selenium IDE not supported on new Firefox versions.

### PR DESCRIPTION
**Pull request by revatij880**
Added updates on Selenium IDE not supported on new Firefox versions. Also updated Selenium Conference broken link on index.html with list of links to all confs till now.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,8 +26,15 @@
 
 <p>Selenium brings together browser vendors, engineers, and enthusiasts
  to further an open discussion around automation of the web platform.
- The project organises <a href="http://www.seleniumconf.org/">an annual conference</a>
```
Comment (shs96c) [inline index.html:29]: It would be better to change this link to point to `http://seleniumconf.com`

File: quick.html [hunk 2]
```diff
@@ -26,6 +26,7 @@
 same application which you push live.
 
 
+
```
Comment (shs96c) [inline quick.html]: I think this got in by mistake :)

File: quick.html [hunk 3]
```diff
@@ -52,8 +53,13 @@
...
-<h2>Selenium Grid</h2>
+<p>The Selenium project team <a href="https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/">announced</a>
+ in August 2017 that Selenium IDE is 
+not compatible with Firefox 55 and future versions of the web browser.
+Hence, Selenium IDE will no longer work. The two main causes were : 
+ Firstly, Firefox is switching extensions from the "XPI" format to more widely 
+ <a href="https://blog.mozilla.org/addons/2016/11/23/add-ons-in-2017/">"Web Extension"</a> mechanism. Secondly, The project lacks time and energy to move 
+ the IDE forward and take advantage of the new technologies.
 
 <p> Soon after development of WebDriver tests, you may face a need of
```
Comment (shs96c) [inline quick.html]: Missing `Selenium Grid` heading.

File: quick.html [hunk 4]
```diff
@@ -52,8 +53,13 @@
 <i>throw away code</i>. We're sure that once you get used to the
 WebDriver API, you will never use Selenium IDE.
 
-
-<h2>Selenium Grid</h2>
+<p>The Selenium project team <a href="https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/">announced</a>
+ in August 2017 that Selenium IDE is 
+not compatible with Firefox 55 and future versions of the web browser.
+Hence, Selenium IDE will no longer work. The two main causes were :
```
Comment (shs96c) [inline quick.html]: I think we can omit the reasons --- if people are interested, they can read the blog post.

File: quick.html [hunk 5]
```diff
@@ -52,8 +53,13 @@
...
 
-
-<h2>Selenium Grid</h2>
+<p>The Selenium project team <a href="https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/">announced</a>
+ in August 2017 that Selenium IDE is 
+not compatible with Firefox 55 and future versions of the web browser.
+Hence, Selenium IDE will no longer work. The two main causes were : 
+ Firstly, Firefox is switching extensions from the "XPI" format to more widely 
+ <a href="https://blog.mozilla.org/addons/2016/11/23/add-ons-in-2017/">"Web Extension"</a> mechanism. Secondly, The project lacks time and energy to move 
+ the IDE forward and take advantage of the new technologies.
```
Comment (shs96c) [inline quick.html]: We should mention that there is a new IDE in development. There's no link right now, but the repo is at `https://github.com/seleniumhq/selenium-ide`

**Review by shs96c — COMMENTED** [review]
Hi!

Thanks for the changes. I've asked for a few alterations, but this is a great start. Thank you!

**Comment by revatij880** [comment]
Hello @shs96c. Thanks a lot for reviewing my pull request! I have made changes based on your review and pushed again for review.

File: quick.html (hunk 5 above)
Comment (vikramvi) [reply quick.html]: @shs96c I hope it gets released sooner :)

**Comment by diemol** [comment]
Hey @revatij880, I am sorry we did not address your PR sooner, please do find the time again to help us with the docs, we will try to address it sooner.

I'll close this PR since the contents it was changing are old and now #210 addresses it.

**Pull request closed without merging**